packages/template/__tests__/keywords/in-element.test.ts

```diff
@@ -7,24 +7,24 @@ declare const element: HTMLElement;
 declare const shadow: ShadowRoot;
 
 // Can be invoked with an element
-emitComponent(inElementKeyword({}, element));
+emitComponent(inElementKeyword(element));
 
 // Can be invoked with a ShadowRoot
-emitComponent(inElementKeyword({}, shadow));
+emitComponent(inElementKeyword(shadow));
 
 // Accepts an `insertBefore` argument
-emitComponent(inElementKeyword({ insertBefore: null }, element));
-emitComponent(inElementKeyword({ insertBefore: undefined }, element));
+emitComponent(inElementKeyword(element, { insertBefore: null, ...NamedArgsMarker }));
+emitComponent(inElementKeyword(element, { insertBefore: undefined, ...NamedArgsMarker }));
```

Comment:
Not related to this PR, but is `undefined` allowed? [The docs][docs] say only `undefined` is.

> It also supports an optional named argument:
> 
> - `insertBefore` -- by default the DOM element's content is replaced when used as `destinationElement`. Passing `null` changes the behaviour to appended at the end of any existing content. Any other value than `null` is currently not supported.

[docs]: https://api.emberjs.com/ember/4.7/classes/Ember.Templates.helpers

Reply:
I wasn't the original author here, but I think at runtime passing `undefined` is treated the same as not specifying `insertBefore` at all. Whether we should allow explicit `undefined` or not is a conversation I'm happy to have (my leaning is to say no), but I'd rather not change the semantics as part of this PR either way.

Reply:
Totally agreed about not changing it in this PR.